tibetegya/node-weconnect
# #162946314 Add user endpoints

#### What does this PR do?
This PR adds Authentication/Authorisation functionality whereby a user is able to 
- Sign up
- Sign In
- Logout
- Verify their account
- Reset their password.
 #### Description of Task to be completed?
- Add user endpoints
 #### How should this be manually tested?
- Clone the repository
- run `npm run start:dev` the server should be running on `localhost:3000`
- user endpoints are on `/api/v1/auth/` for example `localhost:3000/api/v1/auth/`
 #### What are the relevant pivotal tracker stories?
[#162946314](https://www.pivotaltracker.com/story/show/162946314)
